examples/manipulate_sys_nice.rs

```diff
@@ -50,11 +50,17 @@ fn main() {
 
 fn renice(prio: libc::c_int) -> libc::c_int {
     // This is not proper logic, as it does not drain errno.
+    #[cfg(target_arch = "mips")]
+    return unsafe { libc::setpriority(libc::PRIO_PROCESS as i32, 0, prio) };
```

Comment:
Are you building this with a musl libc by chance? According to libc source this should be u32 on all Linux arches (which doesn't match my manpage), but i32 on musl. If so, the cfg conditional should be targeting that (if possible).